CatAnalyzer/scripts/electronSFROOT2Cfg

```diff
@@ -1,30 +1,64 @@
 #!/usr/bin/env python
 
 from ROOT import *
+from math import sqrt
 
-f = TFile("egammaEffi.txt_SF2D.root")
-h = f.Get("EGamma_SF2D")
+def getData(fileName):
+    f = TFile(fileName)
+    h = f.Get("EGamma_SF2D")
 
-nbinsX = h.GetNbinsX()
-nbinsY = h.GetNbinsY()
+    nbinsX = h.GetNbinsX()
+    nbinsY = h.GetNbinsY()
 
-eta_bins = [h.GetXaxis().GetBinLowEdge(i) for i in range(1, nbinsX+2)]
-pt_bins  = [h.GetYaxis().GetBinLowEdge(i) for i in range(1, nbinsY+2)]
+    eta_bins = [h.GetXaxis().GetBinLowEdge(i) for i in range(1, nbinsX+2)]
+    pt_bins  = [h.GetYaxis().GetBinLowEdge(i) for i in range(1, nbinsY+2)]
 
-values = []
-errors = []
-for i in range(1, nbinsX+1):
-    for j in range(1, nbinsY+1):
-        val, err = h.GetBinContent(i,j), h.GetBinError(i,j)
-        values.append(val)
-        errors.append(err)
+    values = []
+    errors = []
+    for i in range(1, nbinsX+1): # eta bins
+        for j in range(1, nbinsY+1): # pt bins
+            val, err = h.GetBinContent(i,j), h.GetBinError(i,j)
+            values.append(val)
+            errors.append(err)
+
+    return [eta_bins[:], pt_bins[:], values[:], errors[:]]
+
+resRECO = getData("effRECO.root")
+resID   = getData("effID.root")
+
+nbins_RECO = len(resRECO[0]), len(resRECO[1])
+nbins_ID   = len(resID[0]), len(resID[1])
+
+eta_bins = sorted(list(set(resRECO[0]).union(set(resID[0]))))
+pt_bins = sorted(list(set(resRECO[1]).union(set(resID[1]))))
+
+values, errors = [], []
+for eta in eta_bins[:-1]:
+    iRECO = next(k for k,v in enumerate(resRECO[0]) if v > eta)-1
+    iID   = next(k for k,v in enumerate(resID[0]) if v > eta)-1
+
+    for pt in pt_bins[:-1]:
+        jRECO = next(k for k,v in enumerate(resRECO[1]) if v > pt)-1
+        jID   = next(k for k,v in enumerate(resID[1]) if v > pt)-1
+
+        addSyst = 0.01
+        if jRECO < 0:
+            addSyst = 0.03
+            jRECO = 0
```

Comment:
For an inline note - it is recommended to assign additional 3% syst. unc. for the low pT region which is not covered by the /uniform/ GSF reco effic SF. For the others, 1% syst. unc is recommended.